### PR TITLE
Fix ambigious description for WattTime account

### PR DESCRIPTION
**Pull request by YaSuenag**
## Summary

Get Started said Python is required for registering to WattTime. However it is not required - we can register in another way.

The essence here is to guide WattTime account registration to SDK users. Python is not important.  
In addition, "WattTime Configuration" which the link points do not mention about Python.

## Changes

Replace to "WattTime account" in Pre-requisites section.

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [ ] Documentation Updates Made?
- [ ] Are there any API Changes? If yes, please describe below.
- [ ] This is not a breaking change. If it is, please describe it below.

## Are there API Changes?

No

## Is this a breaking change?

No

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,4 +33,4 @@ RUN apt-get update && apt-get install jq -y
 
 RUN chmod +x entrypoint.sh
 #ENTRYPOINT ["/CarbonAwareCLI"]
-ENTRYPOINT ["/entrypoint.sh"]
```
Comment (vaughanknight) [inline Dockerfile:36]: Why was this changed as part of the documentation update?  Is is part of a broader change?

Please either:
* Add a description to the PR overview _or_ 
* Remove from the PR as it does not relate to the current PR documentation (preferred).

While it's a small change a general documentation clarification change shouldn't change behaviour (even if small)

**Review by vaughanknight — CHANGES_REQUESTED** [review]
Small update requested.

Documentation clarificaiton/correction changes should avoid having code changes (even if small) to make it easier to trace and reduce risk of system changes alongside documentaiton changes.  Please refer to the comment.

File: Dockerfile (hunk 1 above)
Comment (YaSuenag) [reply Dockerfile]: I do not intent to change any other files - it seems to be caused by merge commit.  
I reverted the change in Dockerfile. Could you review again?

**Review by Willmish — APPROVED** [review]
LGTM
